### PR TITLE
datafusion 6.0.0

### PR DESCRIPTION
**Pull request by jimexist**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Jimexist** [comment]
oops i guess in building ballista it requires `rustfmt` to be present. cc @houqp

File: Formula/datafusion.rb [hunk 1]
```diff
@@ -16,8 +16,14 @@ class Datafusion < Formula
   end
 
   depends_on "rust" => :build
+  # building ballista requires installing rustfmt
+  depends_on "rustup-init" => :build
```
Comment (carlocab) [inline Formula/datafusion.rb:20]: This seems like the wrong way to go about it. We should instead add a `rustfmt` formula and make that a build dependency here.

Comment (Jimexist) [reply Formula/datafusion.rb:20]: ok let me try to do that

Comment (carlocab) [reply Formula/datafusion.rb:20]: Thanks. You can open a separate PR for `rustfmt` to keep things simple here.

Comment (Jimexist) [reply Formula/datafusion.rb:20]: let me try that in https://github.com/Homebrew/homebrew-core/pull/89599

**Comment by Jimexist** [comment]
@houqp I think we need to (in later releases) update the script so that the tagged commit won't have version including `-SNAPSHOT`

**Comment by houqp** [comment]
> oops i guess in building ballista it requires rustfmt to be present. 

Interesting, do you know how it got pulled in? I certainly didn't expect this.

> @houqp I think we need to (in later releases) update the script so that the tagged commit won't have version including -SNAPSHOT

Good call :+1: Will add this to the automation.

File: Formula/datafusion.rb (hunk 1 above)
Comment (Jimexist) [reply Formula/datafusion.rb]: thanks for the advice, this is now updated (and with minimal changes thanks to rustfmt)

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**